### ISSUE TITLE
Nested subtypes do not render without root pattern

### ISSUE THREAD

**Issue by illepic** [pattern-lab/patternlab-node#1056]
I am using Pattern Lab Node `(see screenshot)` on `Mac`, with Node `v10.16.3`, using the twig php renderer.

![image](https://user-images.githubusercontent.com/295127/65010967-949dbb00-d8c6-11e9-8a3c-eb17a454a2e0.png)

##### Expected Behavior

We nest our non-underscore template files in a `demo/` folder, a la:

![image](https://user-images.githubusercontent.com/295127/65053400-c0e62580-d920-11e9-8c4e-9251bd205a6e.png)

Up until the recent uikit update, the `demo/buttons.twig` pattern would render just fine in the UI at `atoms` -> `button` -> `buttons`. 

##### Actual Behavior

No navigation entries are created for button:

![image](https://user-images.githubusercontent.com/295127/65053474-dbb89a00-d920-11e9-82a2-e871a675e740.png)

This "Button" nav item links to non-existent rendered html file.

##### Steps to Reproduce

Create a pattern such that the renderable template file is in a nested folder only, ie:

```
atoms
  button
    _button.twig
    demo
      buttons.twig
```

"Buttons" will not be available in the navigation UI. 

Note: adding a dummy twig file a the root of the pattern results in proper rendering of the navigation. This is how we're circumventing this for now.

**Comment by sghoweri** [comment]
HMMM I might need to pop open the Nav component’s JSX to see what the data looks like when this is busted vs working as expected.

Assuming the Nav data itself is all there, my best guess would be that the nested nav logic doesn’t see any immediate children so it skips rendering the rest of the Nav section (even though grandchildren exist) — or something to that effect.

Thanks for the ticket @illepic!

**Comment by sghoweri** [comment]
@illepic do you know if this issue was also happening in v1.0.3 of UIKit or if it just started happening with 1.0.4?

**Comment by illepic** [comment]
@sghoweri, the most recent version we were using was (`1.0.1`)[https://github.com/phase2/particle/blob/master/package.json#L49], and all of our "plural" patterns within `demo/` rendered great in the nav. I bumped to `1.0.4` last night and noticed this.

**Comment by illepic** [comment]
Noting here that the issue may manifest in this way as well:

![image](https://user-images.githubusercontent.com/295127/65087981-90c47400-d96c-11e9-8c22-5bfafe631af0.png)

TLDR: hidden (underscore) patterns appear to render in the nav when attempting to circumvent the original issue by adding a dummy visible (non-underscore) pattern to the root of the pattern.

**Comment by sghoweri** [comment]
@illepic not sure... I'm trying to reproduce your folder structure in PL PHP using the exact same UIKit code to narrow down where to look.
